api/src/test/java/org/openmrs/api/db/hibernate/HibernateProgramWorkflowDAOTest.java

```diff
@@ -0,0 +1,789 @@
+	 */
+    @Test
+	public void getConceptStateConversionByUuid_shouldReturnNullIfNoSuchCSCExistsWithUUID() {
+		ConceptStateConversion conceptStateConversion = hibernateProgramWorkflowDAO.getConceptStateConversionByUuid("uuid");
+
+		assertNull(conceptStateConversion);
+	}
+
+	/**
+	 * @see org.openmrs.api.db.hibernate.HibernateProgramWorkflowDAO#getPatientProgramByUuid(java.lang.String)
+	 */
+	@Test
+    public void getPatientProgramByUuid_shouldReturnPatientProgramByUUIDIfExists() {
+    	PatientProgram patientProgram = hibernateProgramWorkflowDAO.getPatientProgramByUuid("61ACD71F-989B-4B4A-B00E-0503B756199F");
+    	
+    	assertNotNull(patientProgram);
+    }
+
+	/**
+	 * @see org.openmrs.api.db.hibernate.HibernateProgramWorkflowDAO#getPatientProgramByUuid(java.lang.String)
+	 */
+    @Test
+	public void getPatientProgramByUuid_shouldReturnNullIfNoSuchPatientProgramExistsWithUUID() {
+		PatientProgram patientProgram = hibernateProgramWorkflowDAO.getPatientProgramByUuid("uuid");
+
+		assertNull(patientProgram);
+	}
+
+	/**
+	 * @see org.openmrs.api.db.hibernate.HibernateProgramWorkflowDAO#getProgramByUuid(java.lang.String)
+	 */
+	@Test
+    public void getProgramByUuid_shouldReturnProgramIfItExistsWithTheGivenUUID() {
+    	Program program = hibernateProgramWorkflowDAO.getProgramByUuid("71ACD71F-989B-4B4A-B00E-0503B756199F");
+    	
+    	assertNotNull(program);
+    }
+
+	/**
+	 * @see org.openmrs.api.db.hibernate.HibernateProgramWorkflowDAO#getProgramByUuid(java.lang.String)
+	 */
+    @Test
+	public void getProgramByUuid_shouldReturnNullIfItDoesNotExistWithTheGivenUUID() {
+		Program program = hibernateProgramWorkflowDAO.getProgramByUuid("uuid");
+
+		assertNull(program);
+	}
+
+	/**
+	 * @see org.openmrs.api.db.hibernate.HibernateProgramWorkflowDAO#getState(Integer)
+	 */
+	@Test
+    public void getState_shouldReturnTheProgramWorkFlowStateIfItExistsWithTheParticularID() {
+    	ProgramWorkflowState programWorkflowState = hibernateProgramWorkflowDAO.getState(1);
+    	
+    	assertNotNull(programWorkflowState);
+    }
+
+	/**
+	 * @see org.openmrs.api.db.hibernate.HibernateProgramWorkflowDAO#getState(Integer)
+	 */
+	@Test
+	public void getState_shouldReturnNullIfNoProgramWorkflowExistsWithTheGivenId() {
+		ProgramWorkflowState programWorkflowState = hibernateProgramWorkflowDAO.getState(100);
+
+		assertNull(programWorkflowState);
+	}
+
+	/**
+	 * @see org.openmrs.api.db.hibernate.HibernateProgramWorkflowDAO#getStateByUuid(java.lang.String)
+	 */
+	@Test
+    public void getStateByUuid_shouldReturnTheStateGivenUUIDIfExists() {
+		ProgramWorkflowState programWorkflowState = hibernateProgramWorkflowDAO.getStateByUuid("61ACD71F-989B-4B4A-B00E-0503B957199F");
+		
+		assertNotNull(programWorkflowState);
+    }
+
+	/**
+	 * @see org.openmrs.api.db.hibernate.HibernateProgramWorkflowDAO#getStateByUuid(java.lang.String)
+	 */
+    @Test
+	public void getStateByUuid_shouldReturnNullIfStateWithUUIDDoesNotExist() {
```

Comment:
@akshay-jha  kindly fix the naming to match the camel case pattern ie `getStateByUuid_shouldReturnNullIfStateWithUuidDoesNotExist`

Reply:
Done